runtime/local/src/lib.rs

```diff
@@ -547,13 +557,39 @@ impl pallet_contracts::Config for Runtime {
     type CallStack = [pallet_contracts::Frame<Self>; 31];
     type WeightPrice = pallet_transaction_payment::Pallet<Self>;
     type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
-    type ChainExtension = ();
+    type ChainExtension = LocalChainExtension;
     type DeletionQueueDepth = DeletionQueueDepth;
     type DeletionWeightLimit = DeletionWeightLimit;
     type Schedule = Schedule;
     type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
 }
 
+pub struct LocalChainExtension;
+
+impl ChainExtension<Runtime> for LocalChainExtension {
```

Comment:
@Maar-io the reason you have problem with this is because you're using two different versions of dapps-staking.

One is using `polkadot-v0.9.24` branch and the other one is using your _local_ path.
Change both dapps-staking and dapps-staking-precompiles to point locally and it will work.

Reply:
Thanks @Dinonard that did the trick!!!!